spec/models/embed/envelope_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe Embed::Envelope do
+  include PURLFixtures
+  describe 'to_bounding_box' do
+    it 'should extract a simple array from the xml' do
+      stub_purl_response_with_fixture(geo_purl)
+      @envelope_ng = Embed::PURL.new('1234').ng_xml.xpath('//gml:Envelope', 'gml'=>'http://www.opengis.net/gml/3.2/').first
+      expect(Embed::Envelope.new(@envelope_ng).to_bounding_box()).to eq [["38.298673","-123.387626"],[ "39.399103", "-122.528843"]] 
```

Comment:
Space inside square brackets detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.